client/src/utils/parser/robotCodeToSsotParsing/robotCodeToSsotParsing.test.js

```diff
@@ -35,15 +35,15 @@ const taskSectionAsArray = parser.getRobotCodeAsArray(correctTaskSection);
 const declaredApplications = parser.getApplicationArray(settingsSectionAsArray);
 
 describe('RobotCode to SSOT Parsing Tests', () => {
-  test('ssot contains right robotMetadata', () => {
+  it('ssot contains right robotMetadata', () => {
```

Comment:
This seems to be the only "it" that is not adjusted grammatically. Maybe we change it to something like "checks if ssot contains right robotMetadata"

Reply:
Your proposition is a bit tricky, as one should not refer to the test itself when describing the test. I've changed it to 
` it('contains the correct robotMetadata', () => {` as the test suite description describes what end product (the ssot) results from the test.